redbot/core/utils/views.py

```diff
@@ -11,7 +11,7 @@
 if TYPE_CHECKING:
     from redbot.core.commands import Context
 
-__all__ = ("SimpleMenu", "SetApiModal", "SetApiView")
+__all__ = ("ConfirmView", "SimpleMenu", "SetApiModal", "SetApiView")
```

Comment:
nitpick: we try to keep items in `__all__` in definition order to make it somewhat easier to check if anything is missing in `__all__`:
```suggestion
__all__ = ("SimpleMenu", "SetApiModal", "SetApiView", "ConfirmView")
```